Automattic/wp-calypso
# Pricing page rework: Add product/plan lightbox descriptions.

#### Proposed Changes

This PR adds the specific product descriptions that are displayed in the product/plan lightbox.

It also fixes an issue where the product lightbox descriptions were not displaying for some products, such as for Backup T2, Security T2, Anti-spam, Scan, Search, Boost, and Social.

Completes asana task: 1202796695664022-as-1202979995370873

- I also renamed the functions & properties for getting the product featured descriptions.  They were named/called productFeatured**Text**, which is inconsistent with the product _description_, product short _description_, product lightbox _description_, etc.  So I renamed "productFeatured**Text**" to "productFeatured**Description**", which is more descriptive and consistent with all the other product descriptions.

#### Testing Instructions

**View the regression:**

- Go to https://cloud.jetpack.com/pricing?flags=jetpack/pricing-page-rework-v1
- Click on "More about Backup" link and then select storage option "1 TB".  Notice that the lightbox product description (just under the product title) disappears.
- Click on the "More about <product>" lightbox links for Anti-spam, Scan, Search, Boost, and Social, and notice that the lightbox product descriptions are missing (not displaying).

**Test this PR:**

- Do any one of these
    * Click on Jetpack Cloud live link below and add this to the end of URL `?flags=jetpack/pricing-page-rework-v1`
    * or spin up this PR 
        * Run `git fetch && git checkout add/jetpack-products-lightbox-description`
        * Run `yarn start-jetpack-cloud`
        * Goto `http://jetpack.cloud.localhost:3000/pricing/?flags=jetpack/pricing-page-rework-v1`
     * Click on the "More about <product>" links for all the products/bundles (except not CRM Entrepreneur)
     * Verify all the lightbox product descriptions are displaying, (be sure to select the storage options variants for Backup & Security)
     * You can compare the lightbox product descriptions copy with this **Figma File**: m0y1jHTuDpvMPkFsOJBoNP-fi-3557%3A123824

#### Screenshots:

![Markup 2022-09-14 at 16 08 55](https://user-images.githubusercontent.com/11078128/190253981-f064f849-0657-44e6-ac9d-df4917494e7c.png)

![Markup 2022-09-14 at 16 09 44](https://user-images.githubusercontent.com/11078128/190254002-930cc5bc-ee90-42fb-a87a-83ab3b1ea6c7.png)

![Markup 2022-09-14 at 16 10 39](https://user-images.githubusercontent.com/11078128/190254021-b3b1ee34-3134-4999-84e3-25ab061f0a11.png)

![Markup 2022-09-14 at 16 11 33](https://user-images.githubusercontent.com/11078128/190254071-b3f669b9-bfa3-4aba-ad92-708699351d61.png)

![Markup 2022-09-14 at 16 12 16](https://user-images.githubusercontent.com/11078128/190254083-2183fca5-a271-4b63-8fc5-b71d329b1c37.png)

![Markup 2022-09-14 at 16 13 13](https://user-images.githubusercontent.com/11078128/190254102-39020d0e-da78-4c37-bb85-cc91597225f9.png)

![Markup 2022-09-14 at 16 13 49](https://user-images.githubusercontent.com/11078128/190254119-e52a7b9b-7adb-4ff1-b191-2b3b6e529f3d.png)


![Markup 2022-09-14 at 16 14 38](https://user-images.githubusercontent.com/11078128/190254169-d903d1bf-31ef-4f24-a230-f75fbd27ea54.png)

![Markup 2022-09-14 at 16 15 37](https://user-images.githubusercontent.com/11078128/190254185-67d7d2d3-3f5e-47c3-89fa-c3042a4377d5.png)

![Markup 2022-09-14 at 16 16 20](https://user-images.githubusercontent.com/11078128/190254204-0524abb5-33e3-487e-a2f6-ccdb683a22aa.png)

![Markup 2022-09-14 at 16 16 54](https://user-images.githubusercontent.com/11078128/190254234-f86f914b-ffa2-4d0c-809f-91086e8b1747.png)


#### Pre-merge Checklist







---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202979995370865
  - https://app.asana.com/0/0/1202979995370873
